contracts/contracts/GrantRound.sol

```diff
@@ -0,0 +1,134 @@
+pragma solidity ^0.8.5;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "./GrantRegistry.sol";
+
+contract GrantRound {
+  using SafeERC20 for IERC20;
+  // @notice Unix timestamp of the start of the round
+  uint256 public startTime;
+  // @notice Unix timestamp of the end of the round
+  uint256 public endTime;
+  // @notice Contract owner
+  address public owner;
+  // @notice GrantsRegistry
+  address public registry;
+  // @notice ERC20 token that accepts pool donations
+  IERC20 public donationToken;
+  // @notice URL pointing to grant round metadata (for off-chain use)
+  string public metaPtr;
+  // @notice minimum contribution amt that can be made
+  uint256 public minContribution;
```

Comment:
thoughts on maxing this immutable? 
Highly unlikely this would need to be changed